docker/Dockerfile

```diff
@@ -0,0 +1,92 @@
+# Dockerfile for building a base image with an ETA environment atop a
+# Linux-based OS.
+#
+# ARGs:
+#   BASE_IMAGE
+#   TENSORFLOW_VERSION
+#
+# TensorFlow version notes:
+#   - For CPU-only images, use tensorflow==1.12.0
+#   - For GPU-enabled images, use the TensorFlow version compatible with the
+#     CUDA version you are using:
+#       - CUDA 8: tensorflow-gpu==1.4.0
+#       - CUDA 9: tensorflow-gpu==1.12.0
+#       - CUDA 10: tensorflow-gpu==1.14.0
+#
+# Copyright 2017-2020, Voxel51, Inc.
+# voxel51.com
+#
+# Brian Moore, brian@voxel51.com
+#
+
+#
+# The base image to build from, which must be Linux-based
+#
+
+ARG BASE_IMAGE
+FROM $BASE_IMAGE
+
+#
+# Install ETA
+#
+# Notes:
+#   ETA supports Python 2.7.X or Python 3.6.X
+#
+# `ppa:deadsnakes/ppa` is used in order to install Python 3.6 on Ubuntu 16.04
+# https://askubuntu.com/questions/865554/how-do-i-install-python-3-6-using-apt-get
+#
+# `https://bootstrap.pypa.io/get-pip.py` is used to install pip on Python 3.6
+# https://askubuntu.com/questions/889535/how-to-install-pip-for-python-3-6-on-ubuntu-16-10
+#
+# numpy==1.16.0 is enforced as a last step because tensorflow requires this
+# version to function properly, and some commands here seem to mess with the
+# numpy version installed via the `requirements.txt` file
+#
+
+RUN apt-get update \
+    && apt-get -y --no-install-recommends install software-properties-common \
+    && add-apt-repository -y ppa:deadsnakes/ppa \
+    && apt-get update \
+    && apt-get -y --no-install-recommends install \
+        sudo \
+        build-essential \
+        pkg-config \
+        ca-certificates \
+        cmake \
+        cmake-data \
+        unzip \
+        pciutils \
+        git \
+        curl \
+        wget \
+        python3.6 \
+        python3.6-dev \
+        libcupti-dev \
+        ffmpeg \
+        imagemagick \
+    && ln -s /usr/bin/python3.6 /usr/local/bin/python \
+    && curl https://bootstrap.pypa.io/get-pip.py | python
+
+WORKDIR /dev
```

Comment:
Here's your problem - `/dev` is a `tmpfs`: https://superuser.com/questions/675568/shrink-or-delete-udev-partition-in-ubuntu/675664#675664

Change the working directory something like `/usr/src/app` instead.

Reply:
AHHHHHHH. Thanks for catching